modules/services/vdirsyncer.nix

```diff
@@ -0,0 +1,70 @@
+{ pkgs, lib, config, ... }:
+with lib;
+let
+  cfg = config.services.vdirsyncer;
+  formatToUnknownFormat = settings:
```

Comment:
Looks like the format they use is basically INI but with JSON values... https://github.com/pimutils/vdirsyncer/blob/079a156bf82fe86173d0d05ec0937703b5c6a735/vdirsyncer/cli/config.py#L151

So like,

```nix
settingsFormat = {
  type = types.attrsOf (types.attrsOf (pkgs.formats.json {}).type);
  generate = name: value: pkgs.writeText name (lib.generators.toINI {
    mkKeyValue = lib.generators.mkKeyValueDefault {
      mkValueString = builtins.toJSON;
    } " = ";
  } value);
};
```